src/Lachain.Core/CLI/ConsoleCommands.cs

```diff
@@ -268,16 +270,17 @@ public string SendTransaction(string[] arguments)
         public string VerifyTransaction(string[] arguments)
         {
             var tx = Transaction.Parser.ParseFrom(arguments[1].HexToBytes());
-            var sig = arguments[2].HexToBytes().ToSignature();
+            bool useNewChainId = HardforkHeights.IsHardfork_8Active(_blockManager.GetHeight());
```

Comment:
I think it should be _blockManager.GetHeight() + 1, if it goes to the next block